designdocs/capacity_staking_rewards_economic_model.md

```diff
@@ -0,0 +1,110 @@
+
+## Problem Statement:
+A system consisting only of providers and coinless users who delegate to providers will tend toward centralization.
+To build a self-sustaining Frequency network where control is decentralized, a variety of economic solutions are needed.  One of these is the ability to stake FRQCY token in return for something; this creates an incentive for participation and involvement with the chain fundamentals and governance.
+
+How is that so?  Capacity is how Frequency intends Providers to pay for the vast majority of their on-chain messages. If Providers receive some capacity when end users stake, then they lose capacity if those end users unstake. If a Provider's Capacity from Reward staking is significant, this gives Reward Stakers some power over Providers. If a Provider is utilizing all or nearly all their Capacity almost every Epoch -- which they should be if trying to be economical --  then even a small percentage of lost Capacity will literally cost them to replace it.
+
+We imagine that FRQCY could be airdropped by various entities to encourage onboarding, including Providers themselves. Providers may offer airdrops in exchange for such things as bringing in new users, or sharing links on other platforms, then encourage their users to stake their FRQCY for rewards.  Rewards could be exchanged for in-dApp benefits such as premium features, special emoji, avatar customization, and the like.  Rewards Staking could also unlock features.
+
+## Assumptions
+* The exact formula for calculating rewards is determined in advance and used in the implementation of this design.
+* Rewards are not prorated; they are calculated based on the staking account balance at the start of an Era.
+* Like other blockchains, we round away "dust", values that are _10<sup>-tbd</sup>_ FRQCY or less.
+
+## To be decided:
+* Is Reward Pool token total itself fixed (with input values determined by governance), or is it the Reward _rate_ that is fixed?
+* What are the inputs to the economic model(s) for Capacity generation and Rewards?
+* How does the capacity generated by Maximized Capacity Staking differ from that generated by Rewards Capacity Staking? Maximized is more than Rewards, but by how much?
+* Are rewards compounded?
+
+## Economic Model
+By "economic model" we mean the formulae used to manage staking rewards to achieve the goals of decentralization, economic stabiilty, and sustainability.
+
+Inputs in the economic model to consider could include:
+* Proportion of accounts are staking to total accounts
+* Mean number MSAs per Provider, or the mean compared to median.
+* Proportion of token staked to free balance held by non-Treasury accounts
```

Comment:
The proportion of tokens staked **compared** to **the** free balance held by non-Treasury accounts.